product_attribute_model_link/README.rst

```diff
@@ -0,0 +1,142 @@
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fproduct--attribute-lightgray.png?logo=github
+    :target: https://github.com/OCA/product-attribute/tree/16.0/product_attribute_model_link
+    :alt: OCA/product-attribute
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/product-attribute-16-0/product-attribute-16-0-product_attribute_model_link
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runboat-Try%20me-875A7B.png
+    :target: https://runboat.odoo-community.org/builds?repo=OCA/product-attribute&target_branch=16.0
+    :alt: Try me on Runboat
+
+|badge1| |badge2| |badge3| |badge4| |badge5|
+
+This module allows to link product attributes to models and populate attribute values from the model records and vice versa.
+
+When a record is created in a model that is linked to an attribute new attribute value will be created automatically.
+
+When a record that is linked to an attribute value is deleted linked attribute value will be deleted if it is not used or archived otherwise.
+
+If a related record field value linked to an attribute value is updated the attribute value name is updated accordingly.
+
+If a module that implements a model linked to attribute(s) is uninstalled all the linked attribute values remain in place.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Use Cases / Context
+===================
+
+Sometimes you would like to create product variants based on other model values. 
+
+For example you have a "T-Shirt" product in which two attributes are used among others: "design" and "material".
+
+At the same time you store information about both of them in your db in the dedicated models.
+
+"Design" model keeps the information about image print name, image print category, image author and stores the image file.
+
+"Material" model keeps the information about material name, material type (synthetic/natural), material density and stores a handling instruction in pdf.
+
+Using the regular Odoo flow one will need to create attributes for design and materials and then add values to them.
+
+Eg "Material: cotton, silk, wool", "Design: Fancy Clown, Doge, Pepe, See beach"
+
+And also add the same records to the "Materials" and "Design" models.
+
+Configuration
+=============
+
+Go to the "Inventory/Configuration/Products/Attributes" menu and select an existing or create a new attribute.
+
+Following configuration fields are available:
+
+- Linked Model. Model which records will be used for the attribute values. Cannot be a transient model. Warning: changing or removing existing value will not affect existing attribute values!
+
```

Comment:
This warning appears every time when a new attribute is created. Even if it's not linked to any model. This is really confusing and misguiding. 
It should be shown only when an attribute is linked to a field.
And it should be some message in the form header. But not a pop-up for sure.
![image](https://github.com/OCA/product-attribute/assets/2234999/2e3e2634-bc55-49f8-84d8-68d8066d7b24)